tests/test_kernel_density_estimation_classifier.py

```diff
@@ -0,0 +1,127 @@
+"""Unit tests for the KNN classifier."""
+
+import math
+
+import geomstats.backend as gs
+import geomstats.tests
+from geomstats.geometry.euclidean import Euclidean
+from geomstats.geometry.hypersphere import Hypersphere
+from geomstats.learning.kernel_density_estimation_classifier \
+    import KernelDensityEstimationClassifier
+from geomstats.learning.radial_kernel_functions import triangular_radial_kernel
+
+TOLERANCE = 1e-4
+
+
+class TestKernelDensityEstimationClassifier(geomstats.tests.TestCase):
+    """Class defining the Kernel Density Estimation Classifier tests."""
+
+    def setUp(self):
+        """Define the parameters to test."""
+        gs.random.seed(1234)
+        self.radius = math.inf
+        self.dim = 1
+        self.space = Euclidean(dim=self.dim)
+        self.distance = self.space.metric.dist
+
+    @geomstats.tests.np_only
```

Comment:
About that, the `assignment` docstring needs to be updated (incoming). Basically, you can provide values as a list or an array, and the indices as either a list of tensor positions, e.g. `[(x1, y1), (x2, y2)]`, a list/1D array of booleans along a given axis, or an array of booleans of the same dimension as `x`.

However, since you are working on this part of the backend @ninamiolane , adding this to that PR would mean rebasing and getting quite a number of additional commits, which might make it difficult to review ?

Reply:
For now I have a problem with nose2 on my computer: the tests does not pass on my computer (even the unmodified master branch) and they pass on Travis (for any backend).
So I do not know yet if the tests will pass for pytorch and tensorflow.
Could it be because I use  python 3.8 ? The other requirements looks to be fine, I will check again if necessary.

Reply:
It could be that python3.8 is the problem, as we have configured travis to only run tests with python3.7 for now (to decrease the time travis takes, since we are having a lot of PRs these days).